docs/blog/newsletter-december-2019.rst

```diff
@@ -0,0 +1,127 @@
+.. post:: December 03, 2019
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2019
+#########################################
+
+Hi everybody! It's Beth here, with the last issue of 2019. And what a year it's been! Four conferences (including our first time in `Vilnius </conf/vilnius/2019/>`__, and the recently-wrapped-up `Australia conference </conf/australia/2019/>`__), more meetups than you can shake a stick at, and an ever-busier community on Slack.
```

Comment:
I suspect we don't want to highlight Vilnius as there probably won't be another.

Reply:
fixed!